src/main/java/com/WebController/EventController.java

```diff
@@ -25,20 +32,36 @@
     @Autowired
     private AssociationRepository assoicationRepo;
 
+    @Autowired
+    private UserRepository userRepository;
+
+    @Autowired
+    private ConfirmationTokenRepository confirmationTokenRepository;
+
+    @Autowired
+    private EmailSenderService emailSenderService;
+
+    @Autowired
+    private MessageSender messageSender;
+
     @GetMapping("/")
     public String homePage() {
         return "index";
     }
 
+    /**
+     * Handler for patients creation.
+     * User create a patient in front end and associate patient with multiple events.
+     * Data posted with JSON string and parsed to Event objects directly.
+     * Event duplication should be avoided.
+     */
```

Comment:
shouldn't need "consumes = "application/jason" if it is already annotated as a REST endpoint